ProjectBank/Core/ProjectDTO.cs

```diff
@@ -1,6 +1,24 @@
 ﻿namespace ProjectBank.Core;
 
-public class ProjectDTO
+public record ProjectDTO(int Id, string Title, Status Status, int UserId);
+
+public record ProjectDetailsDTO(int Id, string Title, Status Status, int UserId, string? Description, DateTime CreationDate, DateTime UpdatedDate, IReadOnlyCollection<string> Tags, IReadOnlyCollection<UserDTO> Participants) : ProjectDTO(Id, Title, Status, UserId);
+
+public record CreateProjectDTO
 {
+    [StringLength(50)]
+    public string Title { get; set; }
+
+    [StringLength(500)]
+    public string? Description { get; set; }
+
+    public int UserId { get; set; }
+    public ICollection<string>? Tags { get; set; }
```

Comment:
Maybe some spacing here?
```suggestion
    public int UserId { get; set; }
    
    public ICollection<string>? Tags { get; set; }
```